### ISSUE TITLE
A11y_Azure Data Studio_Nov2023_Database Properties Preview_Filegroups_ScreenReader: NVDA/JAWS announcing irrelevant information when we hit enter key to check the check box present in rows table. 

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/azuredatastudio#25130]
**"Please do not close this bug. This bug should only be closed by Trusted Tester after verification"**
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
## Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #WCAG1.3.1; #FTP; #SH_Azure Data Studio_Nov2023_Win32_Dec2023; #SQL Azure Data Studio; #Win32; #NVDA; #STP; #JAWS; #NonBenckmark;
## Environment Details:
Application Name: Azure Data Studio
Operating System: Win11
Azure data studio 1.47.0
Jaws [2023.2212.23]
NVDA [2023.3.0]
## Repro Steps:
1. Turn on NVDA screen reader
2. Install Azure data studio application and open it
3. Make sure to enable "Preview" feature in settings.
4. Tab till "Database" and press enter to expand the tree.
5. Tab till any database and pressF10 to open context menu.
6. Tab till "Properties Preview" and press enter.
7. Tab till "Filegroups" present in left navigation and press ENTER key
8. TAB till Rows table.
9. Press Right arrow key and navigate to check box and press Space key to select.
10. You will observe that incorrect role as combo box is given for "Enter new name" edit filed.
## Actual Result:
NVDA announcing irrelevant information when we hit enter key to check the check box present in rows table. 
Note: Issue also seen with Jaws.
## Expected Result:
NVDA should announce as checked when we hit enter key to check the check box present in rows table. 
## User Impact:
Screen reader user will not get check box is check or not.
### Attachment:
NVDA attachment
https://github.com/microsoft/azuredatastudio/assets/124254981/668b0801-c498-4dbf-be72-04172fa07d6d
JASW attachment
https://github.com/microsoft/azuredatastudio/assets/124254981/3d21f30e-5ee5-41ef-9b87-d60d6d4934f8
Code snippet
![image](https://github.com/microsoft/azuredatastudio/assets/124254981/9628931a-346b-468a-8402-50286637f277)

**Comment by corivera** [comment]
@kapilvaishna This issue has been fixed in the latest Windows Insiders build of ADS here: https://azuredatastudio-update.azurewebsites.net/latest/win32-x64-user/insider

Can you verify that the issue is resolved?

**Comment by kapilvaishna** [comment]
Issue is not repro in latest Windows Insiders build of ADS here: https://azuredatastudio-update.azurewebsites.net/latest/win32-x64-user/insider , So closing the bug.

NVDA fixed
https://github.com/microsoft/azuredatastudio/assets/124254981/9f8c6ba6-b59d-481f-986d-1f7161385fb5

Jaws Fixed
https://github.com/microsoft/azuredatastudio/assets/124254981/c1abf4a4-03d7-4c83-9cf9-d8dfd42a2435
